tools/LibraryVersionsGenerator/PackageVersionDefinitions.cs

```diff
@@ -115,6 +115,16 @@ internal static class PackageVersionDefinitions
             }
         },
         new()
+        {
+            IntegrationName = "SqlClientSystem",
+            NugetPackageName = "System.Data.SqlClient",
+            TestApplicationName = "TestApplication.SqlClient.System",
+            Versions = new List<PackageVersion>
+            {
+                new("*")
```

Comment:
Add low boundary, to avoid further changes. in case of new version release.
duplication by `*` should be supported by generator. 

```suggestion
                new("4.8.5"),
                new("*")
```